packages/deployment/lib/configValidators.js

```diff
@@ -7,6 +7,10 @@ const get = require('lodash.get');
  *
  */
 function validateWorkflowDefinedLambdas(config) {
+  if (!config.lambdas) {
```

Comment:
I think this defeats the purpose of the validator - it should be if !config.stepFunctions, as that validation routine is checking that you have a lambda defined for each definition in a step function in the configuration.

Reply:
OK, makes sense. I had to add this because it was breaking the deployment of the DataStorage stack, which has no lambdas. But if `!config.stepFunctions` should work equally well